mailpoet/lib/Newsletter/NewsletterValidator.php

```diff
@@ -0,0 +1,104 @@
+<?php declare(strict_types=1);
+
+namespace MailPoet\Newsletter;
+
+use MailPoet\Entities\NewsletterEntity;
+use MailPoet\Services\Bridge;
+use MailPoet\Settings\TrackingConfig;
+use MailPoet\Util\License\Features\Subscribers as SubscribersFeature;
+use MailPoet\Validator\ValidationException;
```

Comment:
I think its ok to use ValidationException here. I would have used one of the PHP built-in exceptions here since we don't need any of the methods that ValidationException provides, but I might be missing something regarding the downsides of using the built-in exceptions.

Reply:
I thought it might be nice to use a MailPoet-specific exception (i.e. one that extends `MailPoet/Exception`) in case we ever handle those separately. It probably doesn't matter too much in this case so I'll just leave it as is.